### DISCUSSION TITLE
keystone with react update query

### DISCUSSION THREAD

**Discussion by fayaz-faiz** [keystonejs/keystone#7901] [Questions]
How to write an update query in react to update the posts in keystone js and how to get that param ID in that query

**Comment by flexdinesh** [comment]
Keystone's GraphQL API is like any other GraphQL API. You would update data using mutations. You can use any graphql client to write your mutation to update a post.

[How to send a mutation request to a GraphQL API using apollo client](https://www.apollographql.com/docs/react/data/mutations)
